bot/processors/pennychat.py

```diff
@@ -286,6 +336,18 @@ def channel_select(self, event):
         penny_chat_invitation.channels = ','.join(selected_channels)
         penny_chat_invitation.save()
 
+    @is_block_interaction_event
+    @has_action_id(PENNY_CHAT_VISIBILITY)
+    def visibility_select(self, event):
+        selected_visibility = event['actions'][0]['selected_option']['value']
+        penny_chat_invitation = PennyChatSlackInvitation.objects.get(view=event['view']['id'])
+        if selected_visibility == '10':
+            penny_chat_invitation.visibility = 10
+            penny_chat_invitation.save()
+        if selected_visibility == '20':
+            penny_chat_invitation.visibility = 20
+            penny_chat_invitation.save()
```

Comment:
Since Slack won't allow you to use integers, could convert this to `penny_chat.visibility = int(selected_visibilty)` and eliminate the conditionals. Not a big deal though.

Reply:
I agree.